package exam8_15.exception;

import java.io.File;

public class ExceptionTest {

	public static void main(String[] args) {
		File f = createFile(args[0]);
		System.out.println(f.getName() + " 파일이 성공적으로 생성되었습니다.");
	}
	
	static File createFile(String FileName) {
		
	}

}
